components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt

```diff
@@ -26,6 +26,11 @@ interface Toolbar {
      */
     var url: String
 
+    /**
+     * Sets/Gets the site security to be displayed on the toolbar.
+     */
+    var siteSecure: Boolean
```

Comment:
In preparation for the future we may want to turn this into an enum. It's likely that we want to show other states here too. :)

Reply:
In that case, it won't be better to use sealed classes ?

Reply:
@Amejia481 I don't see anything that we'd want to hold in the sealed class. The enum value should be good enough.

Reply:
👍